src/python/gjettelek/README.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Lærerveiledning - Gjettelek
+level: 3
+language: nb
+---
+
+
+# Om oppgaven {.activity}
+
+I denne oppgaven skal vi lage et enkelt spill som kalles gjetteleken. Målet er at programmet skal velge et tilfeldig tall mellom 1 og 100, og brukeren skal prøve å gjette tallet. Er tallet for lavt sier programmet gjett høyere, og tilsvarende sier programmet gjett lavere om tallet er for høyt. 
```

Comment:
ville skrevet: _Er tallet du gjetter for lavt så sier programmet gjett høyere_. nå er det litt vanskelig å skjønne hva du mener med *tallet*, ettersom det kan bety både det tallet du skal finne og det tallet du gjetter